ocdaction/ocdaction/templates/challenge/challenge_list.html

```diff
@@ -1,48 +1,107 @@
+                        </div>
+                    </div>
+                </div>
             {% endif %}
-        {% empty %}
-            <ul class="col-xs-6 col-xs-offset-3">
-                <h2>You have no challenges yet!</h2>
-            </ul>
         {% endfor %}
+        
+        {% if sorted_challenges %}
+            <div class="col-lg-12">
+                <div class="ibox float-e-margins">
+                    <div class="ibox-title">
+                        <h3 class="text-align-center">My Challenges</h3>
+                        <div class="ibox-tools">
+                            <a class="button button--secondary" href="{% url 'challenge-add' %}">
+                                <span class="glyphicon glyphicon-plus-sign margin-right-5"></span>Add
+                            </a>
+                        </div>
+                    </div>
+                    <div class="ibox-content">
+                        <table class="table table-striped">
+                            <thead>
+                            <tr>
+                                <th class="ibox-content bold">Level</th>
+                                <th class="ibox-content bold">Challenge</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                                {% for challenge in sorted_challenges %}
+                                    {% if not challenge.in_progress %}
+                                        <tr>
+                                            <td class="text-align-left ibox-content">
+                                                {% get_anxiety_level challenge %}
+                                            </td>
+                                            <td class="text-align-left ibox-content">
+                                                <a href="{% url 'challenge' challenge.id %}">
+                                                    {{ challenge.challenge_name | title }}
+                                                </a>
+                                            </td>
+                                            <td class="ibox-content text-align-right">
+
+                                                {% if another_challenge_in_progress %}
+                                                    <a class="link--disabled">
+                                                        <span class="glyphicon glyphicon-play-circle margin-right-5"></span>
+                                                        Do the challenge
+                                                    </a>
+                                                {% else %}
+                                                    <a href="{% url 'challenge-score-form-new' challenge.id %}">
+                                                        <span class="glyphicon glyphicon-play-circle margin-right-5"></span>
+                                                        Do the challenge
```

Comment:
Also we don't use 'Do the challenge' wording anymore

Reply:
What is the wording instead?